controllers/mondooauditconfig_controller.go

```diff
@@ -94,6 +95,18 @@ func (r *MondooAuditConfigReconciler) Reconcile(ctx context.Context, req ctrl.Re
 		log.Error(err, "Failed to get mondoo")
 		return ctrl.Result{}, err
 	}
+	config := &mondoov1alpha1.MondooOperatorConfig{}
+	if err := r.Get(ctx, types.NamespacedName{Name: mondoov1alpha1.MondooOperatorConfigName}, config); err != nil {
+		if errors.IsNotFound(err) {
+			log.Info("MondooOperatorConfig not found, using defaults")
+		}
+		log.Info(err.Error())
```

Comment:
If the error is anything except for IsNotFound() (which is perfectly valid), then we've encountered an unexpected error. We should just log the unexpected error and exit out of Recincile() to try again. 
```suggestion
		if errors.IsNotFound(err) {
			log.Info("MondooOperatorConfig not found, using defaults")
		} else {
		    log.Error(err, "failed to get MondooOperatorConfig)
		    return ctrl.Result{}, err
		 }
```